tests/pages/apiCommands.js

```diff
@@ -0,0 +1,42 @@
+const { expect } = require('@playwright/test');
+const axios = require('axios');
+
+let boardId;
+
+exports.ApiCommands = class ApiCommands {
+
+    /**
+   * @param {import('@playwright/test').Page} page
+   */
+    constructor(page) {
+        this.page = page;
```

Comment:
Where is this variable used? It doesn't make sense to declare variables if they are not used.